quickstart/resources/views/common/errors.blade.php

```diff
@@ -0,0 +1,16 @@
+<!-- resources/views/common/errors.blade.php -->
+
+@if (count($errors))
+    <!-- Form Error List -->
+    <div class="alert alert-danger">
+        <strong>{!! Form::label(trans('message.error')) !!}</strong>
```

Comment:
phân biệt cho chị `{{ }}` và `{!! !!}`

Reply:
dùng {!! !!} để dữ liệu ko bị đánh cắp còn ngược lại thì {{}} sẽ làm cho dữ liệu bị thất thoát ạ!

Reply:
sự khác biệt chính là {{ }} bên trong nó sẽ ko mã hóa dc các html entities, còn {!! !!} thì ngược lại;
ví dụ : ở message.php - ta đặt như sau : 
'error' => '<strong>Rất tiếc! Đã xảy ra sự cố!</strong>'
-> trong errors.blade.php ta tiến hành gọi đến nó! thì với 

{{ trans(message.error }} -> ta sẽ thu dc như sau :  <strong>Rất tiếc! Đã xảy ra sự cố!</strong>

ngược lại :+1: 

{!! trans(message.error) !!} -> sẽ cho ra dòng chữ : Rất tiếc! Đã xảy ra sự cố! dc format theo thẻ <strong> ;

Reply:
câu trả lời ở trên đúng rồi, còn ví dụ thì sai rồi e ới, đơn giản là {{ }} ko in tag ht
ml ngược lại thì đc okie e

Reply:
thèn git hub nó tự format chị ơiiiiiiiiiiiiiii
![image](https://user-images.githubusercontent.com/16754603/42960593-798643e0-8bb6-11e8-867d-f8d12d32cfe2.png)